### PR TITLE
Feature/project init

### PR DESCRIPTION
**Pull request by Avareco**
Project inited, typescript tryed

### CODE REVIEW

File: env.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```
Comment (hollote) [inline env.d.ts:1]: what does this line do?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "dependencies": {
+    "pinia": "^2.0.23",
+    "primeflex": "^3.3.0",
+    "primeicons": "^6.0.1",
+    "primevue": "^3.20.0",
+    "vue": "^3.2.41",
+    "vue-router": "^4.1.5"
+  },
+  "devDependencies": {
+    "@rushstack/eslint-patch": "^1.1.4",
```
Comment (hollote) [inline package.json:21]: why this dependency is added?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "@types/node": "^16.11.68",
+    "@vitejs/plugin-vue": "^3.1.2",
+    "@vue/eslint-config-prettier": "^7.0.0",
+    "@vue/eslint-config-typescript": "^11.0.0",
+    "@vue/tsconfig": "^0.1.3",
+    "eslint": "^8.22.0",
+    "eslint-plugin-vue": "^9.3.0",
+    "npm-run-all": "^4.1.5",
```
Comment (hollote) [inline package.json:29]: why this dependency is added?

**Review by hollote — CHANGES_REQUESTED** [review]

File: env.d.ts (hunk 1 above)
Comment (Avareco) [reply env.d.ts]: File deleted

File: src/App.vue [hunk 4]
```diff
@@ -0,0 +1,190 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import type { Ref } from "vue";
+
+interface IData {
```
[the remark this answers is not in the document]
Comment (Avareco) [reply src/App.vue]: For testing, but we don't need in it

File: package.json (hunk 3 above)
Comment (Avareco) [reply package.json:27]: dependency deleted

**Pull request merged**